dockers/docker-snmp-sv2/start.sh

```diff
@@ -9,6 +9,17 @@ sonic-cfggen -d -y /etc/sonic/snmp.yml -t /usr/share/sonic/templates/snmpd.conf.
 mkdir -p /var/sonic
 echo "# Config files managed by sonic-config-engine" > /var/sonic/config_status
 
+CURRENT_HOSTNAME=`hostname`
+HOSTNAME=`sonic-cfggen -d -v DEVICE_METADATA[\'localhost\'][\'hostname\']`
+
+if [ "$?" == "0" ] && [ "$HOSTNAME" != "" ]; then
+    echo $HOSTNAME > /etc/hostname
+    hostname -F /etc/hostname
+
+    sed -i "/\s$CURRENT_HOSTNAME$/d" /etc/hosts
```

Comment:
>sed -i "/\s$CURRENT_HOSTNAME$/d" /etc/hosts [](start = 4, length = 43)

This line does not work inside docker container.
ref: [http://blog.jonathanargentiero.com/docker-sed-cannot-rename-etcsedl8ysxl-device-or-resource-busy/](http://blog.jonathanargentiero.com/docker-sed-cannot-rename-etcsedl8ysxl-device-or-resource-busy/)

Could you double check and fix?